test/connectors/des/DesConnectorSpec.scala

```diff
@@ -29,28 +25,38 @@ class DesConnectorSpec extends ITSpec {
   val desConnector = injector.instanceOf[DesConnector]
 
   "find some obligations" in {
-    WireMockResponses.obligationsOk
-    val response: VatObligations = desConnector.getObligations("968501689", LocalDate.now(), LocalDate.now()).futureValue
-    response.obligations.head.obligationDetails.size shouldBe 4
-    Json.toJson(response.obligations.head.obligationDetails.last) shouldBe Json.parse(
-      s"""{
-         | "status": "O",
-         | "inboundCorrespondenceFromDate": "2018-01-01",
-         | "inboundCorrespondenceToDate": "2018-01-31",
-         | "inboundCorrespondenceDateReceived": "2018-04-15",
-         | "inboundCorrespondenceDueDate": "2018-03-07",
-         | "periodKey": "18AA"
+    WireMockResponses.obligationsOk("968501689")
```

Comment:
hmm.

I would split this test in two tests (or two groups of tests).
One test would prove that formats can correctly serialize/deserialize case classes from/to jsons.
Another test would be on a PageSpec level and verify that page has rendered correctly. Such test would go through all layers (controller, service, connector) thus would cover a lot of functionality in one go.

Testing pages should include routes in and routes out which in this project should be fairly easy becaus journeys are pretty linear.

Reply:
I've left the tests but they are simplified due to test data being moved